ext/bcmath/libbcmath/src/doaddsub.c

```diff
@@ -103,7 +103,7 @@ bc_num _bc_do_add(bc_num n1, bc_num n2)
 			 */
 			n1bytes += SWAR_REPEAT(0xF6) + n2bytes + carry;
 			/* If the most significant bit is 0, a carry has occurred. */
```

Comment:
This comment seems to be outdated now?

Reply:
No, but it can be quite confusing so I'll add a few comments.

Reply:
Fixed comments